packages/pagodo/PKGBUILD

```diff
@@ -0,0 +1,44 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=pagodo
+pkgver=77.3485c58
+pkgrel=1
+pkgdesc='Google dork script to collect potentially vulnerable web pages and applications on the Internet.'
+arch=('any')
+groups=('blackarch' 'blackarch-scanner' 'blackarch-recon')
+url='https://github.com/opsdisk/pagodo'
+license=('GPL3')
+depends=('python' 'python-beautifulsoup4' 'python-google' 'python-numpy' 'python-requests')
+makedepends=('git')
+source=("git+https://github.com/opsdisk/$pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
+package() {
+  cd $pkgname
+
+  install -dm 755 "$pkgdir/usr/bin"
+  install -dm 755 "$pkgdir/usr/share/$pkgname"
+
+  install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" README.md
+  install -Dm 644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
+
+  rm README.md LICENSE
+
+  cp -a * "$pkgdir/usr/share/$pkgname/"
+
+  cat > "$pkgdir/usr/bin/$pkgname" << EOF
+#!/bin/sh
+cd /usr/share/$pkgname
+exec python pagodo.py "\$@"
+EOF
+
+  chmod +x "$pkgdir/usr/bin/$pkgname"
+}
```

Comment:
We don't need to use the `cat > "$pkgdir/usr/bin/$pkgname"` hack in this case, we can just `install` `ghdb_scraper.py` and `pagodo.py` to /usr/bin, `install -Dm 755 -t pagodo.py ghdb_scraper.py "$pkgdir/usr/bin/"` should work, then copy the other things to `/usr/share/$pkgname` as usual.

Reply:
Sure. I will adjust. Thanks

Reply:
updated from chat applied. Thanks !